model-optimizer/extensions/front/tf/mvn_unrolled.py

```diff
@@ -56,7 +56,8 @@ def pattern(self):
     def replace_sub_graph(graph: Graph, match: dict):
         mvn = MVN(graph, dict(
             name=match['truediv'].name + '/MVN_',
-            required_reduction_indices=[1, 2] if graph.graph['layout'] == 'NHWC' else [2, 3]
+            eps_mode='outside_sqrt',
+            normalize_variance=1
         ))
         mvn.attrs['old_infer'] = mvn.attrs['infer']
         mvn.attrs['infer'] = __class__.infer
```

Comment:
Ideally, we need to refactor this code to avoid this approach.

Reply:
We already discussed that with @mvafin and we agreed to keep it as is. But I agree with you that it should be refactored. But since there are a lot of code changes and there are some technical obstacles to do this we have decided to keep it as is.

Reply:
We decided to not do it on this PR